addons/website_event/views/event_templates_list.xml

```diff
@@ -301,9 +302,20 @@
                         <div class="d-block h-100 w-100">
                             <t t-call="website.record_cover">
                                 <t t-set="_record" t-value="event"/>
-                                <!-- Short Date -->
                                 <meta itemprop="startDate" t-att-content="event.date_begin.isoformat()"/>
-                                <div t-attf-class="o_wevent_event_date position-absolute shadow-sm o_not_editable #{(not opt_events_list_columns) and 'left'} ">
+                                <!-- Short date + Country Flag -->
+                                <div t-if="opt_events_event_location" t-attf-class="o_wevent_date_with_flag o_wevent_event_date position-absolute shadow-sm o_not_editable #{(not opt_events_list_columns) and 'left'} ">
+                                    <div class="o_wevent_event_date_text">
+                                        <span t-out="event.date_begin" t-options="{'widget': 'datetime', 'tz_name': event.date_tz, 'format': 'LLL'}" class="o_wevent_event_month"/>
+                                        <span t-out="event.date_begin" t-options="{'widget': 'datetime', 'tz_name': event.date_tz, 'format': 'dd'}" class="o_wevent_event_day oe_hide_on_date_edit"/>
+                                    </div>
+                                    <div class="o_wevent_event_country_flag">
+                                        <img t-if="event.country_id" t-att-src="event.country_id.image_url" t-att-alt="event.country_id.name"/>
```

Comment:
There appears to be a couple country for which we do not have a flag (see "NO_FLAG_COUNTRIES"), I'm guessing we don't want the placeholder image nor the camera image for those?